mirri/io/parsers/mirri_excel.py

```diff
@@ -51,11 +53,11 @@ def excel_dict_reader(path, sheet_name, mandatory_column_name=None):
         data = dict(zip(header, values))
 
         if mandatory_column_name is not None and not data[mandatory_column_name]:
-            msg = f"Exiting before end of sheet {sheet_name} ends.\n"
-            msg += f"Mandatory column ({mandatory_column_name}) empty. \n"
-            msg += "Check file for empty lines"
-            print(msg)
-            break
+            # msg = f"Exiting before end of sheet {sheet_name} ends.\n"
+            # msg += f"Mandatory column ({mandatory_column_name}) empty. \n"
+            # msg += "Check file for empty lines"
+            # print(msg)
+            continue
```

Comment:
are you sure? with the _continue_ it should skip the current line and process the next one. With _break_ if an row has the mandatory column empty, it'll stop processing the rest of the file, which is not what is supposed to happen.

Reply:
The idea of this piece of code is to stop reading the sheet. It usually happens when you have a excel file, removes some row content(Not the row itself) and when you use this function to parse the file it returs a bunch of empty lines. To try to avoid this I wrote this. 
The difference is that you can have blanck rows between valid rows and my code does not allow blanck rows.

Reply:
But suppose you have something like:

Valid row
Valid row
Valid row
Invalid row
Valid row
Valid row

if you stop reading the excel sheet you will be discarding the last two valid rows, won't you?

Reply:
Yes. This is why I added the message on the screen. As this is an unexpected behavior(having empty lines) I prefer to show an explicit message and stop processing the sheet.

Tomorrow we can discuss it :)

Reply:
Fix: add a message before continue to report the error